openreview/conference/matching.py

```diff
@@ -361,23 +361,23 @@ def _build_config_invitation(self, scores_specification):
                         'description': 'Title of the configuration.',
                         'order': 1
                     },
-                    'max_users': {
+                    'user_demand': {
                         'value-regex': '[0-9]+',
                         'required': True,
-                        'description': 'Max number of reviewers that can review a paper',
+                        'description': 'Max number of users that can review a paper',
```

Comment:
Can we remove `max` from the description?

Reply:
Done.